src/games/strategy/triplea/ai/proAI/ProNonCombatMoveAI.java

```diff
@@ -249,7 +249,7 @@ private void findUnitsThatCantMove(final Map<Territory, ProPurchaseTerritory> pu
 
       // Add max defenders that can be purchased to each territory
       for (final Territory t : moveMap.keySet()) {
-        if (ProMatches.territoryHasInfraFactoryAndIsNotConqueredOwnedLand(player, data).match(t)) {
```

Comment:
Pls double check this method is still used, it's specific enough I could see it being a single usage method.